tokenserver-auth/src/oauth/verify.py

```diff
@@ -16,4 +16,6 @@ def verify_token(self, token):
             # Serialize the data to make it easier to parse in Rust
             return json.dumps(token_data)
         except (ClientError, TrustError):
```

Comment:
Might we want the added context from the traceback, provided by re-raising the exception?
```suggestion
        except (ClientError, TrustError) as error:
            # XXX: debugging
            #return None
            raise error
```

Reply:
This will only give us an extra line in the traceback showing the reraise line, which doesn't add much for this -- if we see `Client/TrustErrors` emitted from `verify_token` we know they've been through this reraise.

The change to reraise is actually a bit silly: it makes the entire try/except clause pointless. If this code was intended to live longer (I believe it will be reverted/changed soon) I would have removed the entire clause instead -- this is just a quick temporary change.